modules/reports/models/scheduled_reports.php

```diff
@@ -234,11 +234,11 @@ static public function edit_report($id=false, $rep_type=false, $saved_report_id=
 	 */
 	static function update_report_field($id=false, $field=false, $value=false)
 	{
-		$id = (int)$id;
-		$field = trim($field);
-		$value = trim($value);
 		$db = Database::instance();
-		$sql = "UPDATE scheduled_reports SET ".$field."= ".$db->escape($value)." WHERE id=".$id;
+		$id = (int)$id;
+		$field = $db->escape_column(trim($field));
+		$value = $db->escape(trim($value));
+		$sql = "UPDATE scheduled_reports SET $field=$value WHERE id=$id";
```

Comment:
I think we should have curly braces around the variables here (or use concatenation). While I guess it works in this case, I would be worried that PHP would look for a variable named `$field=`.

Reply:
Shouldn't the `$` be inside the curly bracket? :thinking:

Perhaps that works, never seen it like that before though. I don't see a reference to that syntax on: https://www.php.net/manual/en/language.types.string.php#language.types.string.parsing.complex 

Sorry if nitpicking!

Reply:
Ah, you're most probably right! My mistake for thinking I know a bit of PHP by now. :)